manifests/course/puppetize.pp

```diff
@@ -14,6 +14,10 @@
       mode  => '0644',
     }
 
+    class { 'classroom':
```

Comment:
This pulls in a *ton* of dependent classes. I'd rather not if we can avoid it. Did you try just `classroom::virtual`?